src/test/java/seedu/address/logic/commands/getcommands/GetMedicationCommandTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands.getcommands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.BENSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.GetCommand;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.MedicationContainsKeywordsPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code GetMedicationCommand}.
+ */
+public class GetMedicationCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        MedicationContainsKeywordsPredicate firstPredicate = new MedicationContainsKeywordsPredicate(
+                Collections.singletonList("paracetamol"));
+        MedicationContainsKeywordsPredicate secondPredicate = new MedicationContainsKeywordsPredicate(
+                Collections.singletonList("ibuprofen"));
```

Comment:
might want to pull in these medication names as the static variables defined in PastAppointmentBuilder for better abstraction.

Reply:
I think we can rework this issue of pulling the right constants from a new `TestUtilMessages` class. I will provide a temporary fix for this for now.